### DISCUSSION TITLE
Fixed TWDT / deadlock in AsyncTCP when sending long responses

### DISCUSSION THREAD

**Discussion by mathieucarbou** [tbnobody/OpenDTU#2456] [General]
Hello @tbnobody, @schlimmchen, 

I don't know if you remember this issue:  https://github.com/tbnobody/OpenDTU/issues/1705

(I was not able to comment on it).

Turns out the past days / weeks we were able to pinpoint why AsyncTCP was crashing (_or dead-locking when the TWDT was not used_), on long responses or long chunk responses. 

There has been a lengthy discussion around this problem here: https://github.com/mathieucarbou/ESPAsyncWebServer/discussions/165

A fix was implemented and tested: (_we could call that a workaround or patch though because the real fix would be to get rid of the current queue design_)

- https://github.com/mathieucarbou/AsyncTCP (3.2.15)
- https://github.com/mathieucarbou/ESPAsyncWebServer ( 3.4.0)

There is also some perf tests available in my fork and in the discussion above.

**Comment by tbnobody** [comment]
Thanks for the hint!! Will include it in the next release.
